crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -137,9 +137,9 @@ pub enum EcmascriptModuleAssetType {
     /// Module with EcmaScript code
     Ecmascript,
     /// Module with TypeScript code without types
-    Typescript,
+    Typescript { tsx: bool },
     /// Module with TypeScript code with references to imported types
-    TypescriptWithTypes,
+    TypescriptWithTypes { tsx: bool },
```

Comment:
Should we convert `WithTypes` into a Boolean field too?

Reply:
Was wondering if there's a good reason it isn't